std/conv.d

```diff
@@ -1739,7 +1749,22 @@ T toImpl(T, S)(S value)
 }
 
 /// ditto
-T toImpl(T, S)(S value, uint radix)
+T toImpl(T, S)(S value, AllowUnderscores uscores)
+    if ( isExactSomeString!S && isDynamicArray!S &&
+        !isExactSomeString!T && is(typeof(parse!T(value))))
```

Comment:
Hm... probably want that last template constraint to be `is(typeof(parse!T(value, AllowUnderscores.no)))`